recipes/_setup_python.rb

```diff
@@ -18,6 +18,13 @@
   python_version node['cfncluster']['python-version']
 end
 
+if node['platform'] == 'centos' && node['platform_version'].to_i < 7
+  # CentOS 6 - install a newer version of Python using pyenv and make it globally available
```

Comment:
please update changelog to call this out explicitly

Reply:
done